django_app/redbox_app/redbox_core/migrations/0046_chat_archived.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 5.1.1 on 2024-09-26 12:07
```

Comment:
pls can we delete both migrations, and regenerate them? the reason being that the first one will fail (for the usual reason of trying to add a non null field) even though this is immediately fixed by the second one